### DISCUSSION TITLE
Problem with ltsp image

### DISCUSSION THREAD

**Discussion by mdenis35** [ltsp/ltsp#872] [General] [answered]
Hello,

Since two days, i can't renew my ltsp image in ubuntu.

Il do as always and when il put the command :" ltsp image ubuntu.img" as usually,

i take this error :
 
/usr/sbin/ltsp: 1604: eval: Syntax error: word unexpected (expecting ")")

Any idea ?

Thanks.

Marc

**Comment by alkisg** [comment, marked as answer]
Check your /etc/ltsp/ltsp.conf for syntax errors, for example missing quotes.

**Reply by mdenis35** [reply]
Thank's a lot

That's good.

thanks again for your help and your investment in the service of educational establishments of all nations

Marc (France)

**Reply by mdenis35** [reply]
it was effectively an error in the /etc/ltsp/ltsp.conf
